Assets/Scripts/Core/GameController.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace OperationBlackwell.Core {
 	public class GameController : MonoBehaviour {
+		public static GameController Instance {get; private set;}
```

Comment:
Coding style: add a space before get and after private set; in this file

Reply:
The style this is currently in matches the style used in other files, and is thus correct.

Reply:
Interesting as the first PR where these kind of statements are used is #12 and there it is with extra spaces.